src/main/java/core/basesyntax/model/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.model;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final int base1;
+    private final int base2;
```

Comment:
base1, base2 - bad naming for variables
```suggestion
    private final int base1;
    private final int base2;
```